Zotero/Extensions/UIViewController+Extensions.swift

```diff
@@ -69,4 +69,8 @@ extension UIViewController {
         // Parent also didn't return a scene. Trying presenting view controller.
         return presentingViewController?.scene
     }
+
+    var sessionIdentifier: String? {
```

Comment:
All `UIViewController`s which support multiple items controls also contain the `UIButton` for triggering the deferred `UIMenu` and observing of item count. I know the UI will change to tabs at some points, but it might be worth putting those two into this extension or some new extension which would handle the tabbing UI.

I think we even mentioned this at some point, so just to keep that in mind.